src/utils/write-ast.js

```diff
@@ -14,9 +13,6 @@ module.exports = async (filePath, ast) => {
     // https://github.com/benjamn/recast/issues/534
     .replace(/\r?\n\r?\n/g, os.EOL)
 
-  // ensure the directory exists
```

Comment:
Why is this no longer needed?

Reply:
The [outputFile](https://github.com/jprichardson/node-fs-extra/blob/master/docs/outputFile.md) function can create the parent directory automatically if not exists.